components/atom/badgeCounter/src/index.js

```diff
@@ -1,11 +1,15 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
+import React, {Component} from 'react'
 
 class BadgeCounter extends Component {
   render () {
     return (
       <div className='sui-BadgeCounter'>
-        <span className={`sui-BadgeCounter--${this.props.size} sui-BadgeCounter-${this.props.type} sui-BadgeCounter-${this.props.type}--${this.props.thickness}`} />
+        <span
+          className={`sui-BadgeCounter--${this.props.size} sui-BadgeCounter-${
+            this.props.type
```

Comment:
this looks kind of strange but maybe it should be implemented otherwise

Reply:
line limit again.